Week1/homework.js

```diff
@@ -0,0 +1,70 @@
+var mysql = require('mysql');
+var connection = mysql.createConnection({
+  multipleStatements: true,
+  host: 'localhost',
+  user: 'hyfuser',
+  password: 'hyfpassword',
+});
+
+connection.connect();
+const databaseWorld = 'CREATE DATABASE IF NOT EXISTS world';
```

Comment:
You can use 'new_world' instead of 'world' here to use the data being loaded by world.sql